content/news/042/index.md

```diff
@@ -337,7 +337,7 @@ New features from January's [sprint 22][graphite-sprint-22]:
   whole graph each time an edit is made or a value changes. This makes iteration
   faster and enables caching of intermediate computations for faster rendering.
 
-Later this month, the Alpha Milestone 2 release will launch with new node graph
+And soon, the Alpha Milestone 2 release will launch with new node graph
```

Comment:
I made a minor amendment to the previous edition to correct a detail which changed between submission and publication. It seems minor and thus relatively harmless so I hope that's ok.